src/CMakeLists.txt

```
@@ -124,6 +120,10 @@ else()
     endif()
 endif()
 
+# Generate a .pc file
+set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
```

Nitpick as this was not added in this PR but I noticed it know: in general it is not a good idea to make variable that depend on `${CMAKE_INSTALL_PREFIX}` cache variables, as this means that if you modify `CMAKE_INSTALL_PREFIX`, the `INSTALL_PKGCONFIG_DIR` is not automatically updated.
```suggestion
set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
```

---

@traversaro thanks for noticing!  I can add a commit, if you agree.

---

Done in https://github.com/ami-iit/openvr/pull/1/commits/76389e92b1dbb9018f0fff06341bd2a5cc786f26